commonknowledge/awasqa-block-theme
# feat: add import command and fix related bugs

## Description
Adds command `wp import_awasqa`

## Motivation and Context
Imports data from awasqa.org.xml.

## How Can It Be Tested?
1. Take a database backup
2. Run the command `wp import_awasqa`
3. Check the users, organisations, media and posts that have been imported.

## How Will This Be Deployed?
As "How can be tested" above, but on the staging site.
